### DISCUSSION TITLE
Roadmap Info

### DISCUSSION THREAD

**Discussion by avinashdesireddy** [liqotech/liqo#931] [Q&A] [answered]
Hi there, I'm interested in the roadmap for this project. How can I find it?

**Comment by palexster** [comment, marked as answer]
Hi @avinashdesireddy, we haven't updated the README.md yet (in progress), but I can provide you with some information.

We plan to have a v0.3.1 in the following weeks, including the OpenShift support (#914) and several refactoring.

For the next v0.4, we are still discussing about the features we would like to include. Any feedback or suggestion is well appreciated.

**Reply by avinashdesireddy** [reply]
Got it. Thank you @palexster
